documentation/src/main/asciidoc/topics/scripting.adoc

```diff
@@ -1,106 +0,0 @@
-[[scripting]]
```

Comment:
is there new docs for scripts? or it is no longer supported?

Reply:
@pruivo yes, this is an old doc that was in the Server Guide and has been replaced. I forgot to remove it when I made those changes. see: https://raw.githubusercontent.com/infinispan/infinispan/master/documentation/src/main/asciidoc/stories/assembly_server_scripts.adoc